packages/harper-ls/package.yaml

```diff
@@ -0,0 +1,45 @@
+---
+name: harper-ls
+description: The Grammar Checker for Developers
+homepage: https://github.com/chilipepperhott/harper-ls
+licenses:
+  - Apache-2.0
+languages:
+  - Markdown,
```

Comment:
just a heads-up: the trailing commas are unnecessary, in fact they just make the display in the mason window a bit uglier:
![Pasted image 2024-03-05 at 11 10 25@2x](https://github.com/mason-org/mason-registry/assets/73286100/29f442ec-be47-491c-873b-4212a2cbf342)

They don't make the package invalid though, just an visual issue